### PR TITLE
Fix link on popup

### LINKED ISSUE #16: link in popup is broken
it always open last CL's url.

I'll take a look.

### PR DESCRIPTION
**Pull request by sunglim**
Fixes https://github.com/yperess/gerrit-chrome-extension/issues/16

The core change is
`url: BASE_URL + '/' + change._number` -> `url: BASE_URL + '/' + this.textContent`

and style fix.

### CODE REVIEW

File: extension/popup.js [hunk 1]
```diff
@@ -60,13 +58,14 @@ function initUI(items) {
 
     var a = document.createElement('a');
     a.href = '#';
+	var BASE_URL = localStorage['api_endpoint'];
```
Comment (yperess) [inline extension/popup.js]: nit: indent?

File: extension/popup.js [hunk 2]
```diff
@@ -60,13 +58,14 @@ function initUI(items) {
...
     a.addEventListener('click', function(e) {
-        chrome.tabs.update(null, {
-            url: BASE_URL + '/' + change._number
-          });
+      chrome.tabs.update(null, {
+          url: BASE_URL + '/' + this.textContent
       });
+    });
 
-    a.textContent = 'cl/' + change._number;
```
Comment (yperess) [inline extension/popup.js:69]: Please keep 'cl/' this is a part of another feature I'm intending to add, sorry for not writing up the issue before, see #18.

Instead why don't we do:

``` javascript
a.addEventListener('click', function(e) {
  chrome.tabs.update(null, {
    url: BASE_URL + '/' + this.getAttribute('change-id');
  });
})
a.setAttribute('change-id', change._number)
```

**Review by yperess — CHANGES_REQUESTED** [review]

**Comment by sunglim** [comment]
I will fix at office (From home i'm not able to connect to company gerrit. so, i can't test).

File: extension/popup.js (hunk 1 above)
Comment (sunglim) [reply extension/popup.js]: Done.

File: extension/popup.js (hunk 2 above)
Comment (sunglim) [reply extension/popup.js:69]: Done.

**Review by sunglim — COMMENTED** [review]
Thanks for the review

**Comment by sunglim** [comment]
PTAL.

**Pull request merged**